### DISCUSSION TITLE
Questions about Build and Run

### DISCUSSION THREAD

**Discussion by mahalrs** [Azure/orkestra#293] [Q&A]
Whenever types (api/xxx_types.go) are changed, do I need to run both `make generate` and `make manifests` or just the `make generate`.

**Comment by nitishm** [comment]
both the targets - one does the type generations and the other generates the CRD yaml files.

**Reply by mahalrs** [reply]
Thanks

**Reply by nitishm** [reply]
it might be a good idea to stick under a top level make target if you want to try that out. Another thing to careful about is the controller-gen and controller-runtime versions.

**Reply by mahalrs** [reply]
How do I check if Orkestra deployment is using my local docker registry instead of azureorkestra/orkestra.
